### ISSUE TITLE
AirDrop

### ISSUE THREAD

**Issue by Alex-V2000** [dkoluris/ZEPHYRUS-M15-GU502-OpenCore#3]
Hey man, did you test airdrop and can you send files?

**Comment by koluris** [comment]
![Airdrop](https://user-images.githubusercontent.com/31432553/106435997-3d2cca80-647c-11eb-82f6-c6b65bc7e98e.png)

Hmm, mixed feelings about this. I was able to detect all my other devices, file sharing seems to not work with current version of AirportItlwm.

**Comment by Alex-V2000** [comment]
yeah same problem 😂, waiting to fix by new vresion on AirportItlwm
